src/bp/ui-admin/src/telemetry/telemetry_type.d.ts

```diff
@@ -0,0 +1,34 @@
+export interface StoreInfoBody {
+  storedInfo: string
+  loadInfo: Function
+}
+
+export interface StoreInfoType {
+  [key: string]: StoreInfoBody
+}
+
+export interface EventPackageInfoType {
+  [key: string]: EventPackageInfoBody
+}
+
+export interface EventPackageInfoBody {
+  locked: boolean
+  timeout: string
+  getPackage: Function
+}
+
+export interface EventData {
+  schema: string
+  [key: string]: any
+}
+
+export interface TelemetryPackage {
+  schema: string
+  uuid: string
+  timestamp: string
+  bp_release: string
+  bp_license: string
+  event_type: string
+  source: string
+  event_data: EventData
+}
```

Comment:
I can see that only 3 types are used and they're used in a single file (client.ts) please move all of them 3 type declaration in that file directly

Reply:
Fixed in : e135fcf